features/FEATURE_BLE/targets/TARGET_CORDIO/mbed_lib.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "cordio",
+    "config": {
+        "cordio-cfg-max-connections": {
+            "help": "Maximum number of connections",
+            "value": 3,
+            "macro_name": "CORDIO_CFG_MAX_CONNECTIONS"
```

Comment:
My assumption is using macro_name to overwrite the default one - length ? Or why?